metrics/collectors/economic/economic.go

```diff
@@ -110,6 +108,17 @@ func (ec *Collector) Describe(ch chan<- *prometheus.Desc) {
 	// ch <- ec.delegatedShares
 }
 
+func (ec *Collector) cacheOperatorIdIfNotCached() error {
```

Comment:
`cacheOperatorIdIfNotCached` - it's a weird name. don't `cache` already mean if some key is not present it would cache it?

Reply:
Not in this case because we only cache the very first time this is called. If it's already cached we don't update the cache.

Reply:
maybe `maybeCacheOperator`? or `cacheOperatorIfNeeded`? feel free to find another name, but I don't think `cacheOperatorId` works

Reply:
Renamed to `initOperatorId` as we discussed on slack: [fa42170](https://github.com/Layr-Labs/eigensdk-go/pull/54/commits/fa42170176c0d4c501ba4fe192d8704e20ccad9f)

Reply:
IMO just something like `getOperatorID()` will be good. The cache is internal detail/optimization that the caller doesn't need to care about.